specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -541,6 +541,12 @@
             "schema": {
               "$ref": "#/definitions/SuppressionContract"
             }
+          },
+          "404": {
+            "description": "Suppression name is not specified or Suppression Id not found.",
+            "schema": {
+              "$ref":"#/definitions/ARMErrorResponseBody"
+            }
```

Comment:
We are being flagged by S360 that our API would return 404 response code but we haven't define it in our Swagger. So I'm adding 404 response code here. https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=46FCA6B0-7DA3-4527-8EE9-DADDF502FFAF

Reply:
If so, I suggest you add `default` in every response rather than a single `404`. Otherwise, the other error response would also not appear in swagger, like `400`, `409`, `412`...

Reply:
You meant add default to our API controller or Swagger? If Swagger do you have an example for it?

Reply:
You could see https://github.com/azure/azure-rest-api-specs/blob/a3733cc/specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json#L49-L62

Reply:
Got it. For now we don't have a use case for other error code. Could we proceed with single 404 for now?